PDF extractor.py

```diff
@@ -0,0 +1,123 @@
+import fitz
+import re
+import requests
+import webbrowser
+from pathlib import Path
+import argparse
+import sys
+import os
+import pyfiglet
+
+
+
+class PDFExtractor:
+
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+        
+    def pdf_text_reader(self):
+        try:
+            with fitz.open(self.file_path) as pdf:
+                # Gets the pages range that the user wants to print
+                pages_range = pdf.pages(int(input("The page to start from: ")),int(input("The page to end with: ")))
+                text = chr(12).join([page.get_text() for page in pages_range])
+                print(text)
+        except FileNotFoundError:
+            print(f"File not found, please make sure it's the right path: {file_path}")
+
+
+    def pdf_file_extractor(file_path):
+        try:
+            with fitz.open(file_path) as pdf:
+                embedded_files = pdf.embfile_count()
+                for emb_file in range(embedded_files):
+                    file_name = "extracted"
```

Comment:
Get the file name by using embfile_info() or embfile_names()
https://pymupdf.readthedocs.io/en/latest/document.html#Document.embfile_info
https://pymupdf.readthedocs.io/en/latest/document.html#Document.embfile_names